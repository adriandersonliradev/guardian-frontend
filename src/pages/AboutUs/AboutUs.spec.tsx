import { render } from "@testing-library/react";
import { AboutUs } from ".";

describe("AboutUs", () => {
  it("should render correctly", () => {
    const wrapper = render(<AboutUs />);

    expect(wrapper).toMatchSnapshot();
  });
});
