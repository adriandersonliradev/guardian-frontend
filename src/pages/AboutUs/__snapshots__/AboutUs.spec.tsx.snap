// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutUs > should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="nav navbar navbar-expand-lg navbar-light"
      >
        <div
          class="container"
        >
          <a
            class="poppins-semibold navbar-brand"
            href="/"
          >
            <img
              alt=""
              class="d-inline-block"
              height="60"
              src="/src/assets/logo.png"
              width="60"
            />
             
            O Guardi찾o
          </a>
          <button
            aria-controls="responsive-navbar-nav"
            aria-label="Toggle navigation"
            class="navbar-toggler collapsed"
            type="button"
          >
            <span
              class="navbar-toggler-icon"
            />
          </button>
          <div
            class="justify-content-end navbar-collapse collapse"
            id="responsive-navbar-nav"
          >
            <div
              class="navbar-nav"
            >
              <a
                class="nav-link"
                data-rr-ui-event-key="/"
                href="/"
              >
                Home
              </a>
              <a
                class="nav-link"
                data-rr-ui-event-key="tipos-documentais"
                href="tipos-documentais"
              >
                Tipos Documentais
              </a>
              <a
                class="nav-link"
                data-rr-ui-event-key="documentos"
                href="documentos"
              >
                Documentos
              </a>
              <a
                class="nav-link"
                data-rr-ui-event-key="sobre-nos"
                href="sobre-nos"
              >
                Sobre n처s
              </a>
            </div>
          </div>
        </div>
      </nav>
      AboutUs
    </div>
  </body>,
  "container": <div>
    <nav
      class="nav navbar navbar-expand-lg navbar-light"
    >
      <div
        class="container"
      >
        <a
          class="poppins-semibold navbar-brand"
          href="/"
        >
          <img
            alt=""
            class="d-inline-block"
            height="60"
            src="/src/assets/logo.png"
            width="60"
          />
           
          O Guardi찾o
        </a>
        <button
          aria-controls="responsive-navbar-nav"
          aria-label="Toggle navigation"
          class="navbar-toggler collapsed"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="justify-content-end navbar-collapse collapse"
          id="responsive-navbar-nav"
        >
          <div
            class="navbar-nav"
          >
            <a
              class="nav-link"
              data-rr-ui-event-key="/"
              href="/"
            >
              Home
            </a>
            <a
              class="nav-link"
              data-rr-ui-event-key="tipos-documentais"
              href="tipos-documentais"
            >
              Tipos Documentais
            </a>
            <a
              class="nav-link"
              data-rr-ui-event-key="documentos"
              href="documentos"
            >
              Documentos
            </a>
            <a
              class="nav-link"
              data-rr-ui-event-key="sobre-nos"
              href="sobre-nos"
            >
              Sobre n처s
            </a>
          </div>
        </div>
      </div>
    </nav>
    AboutUs
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
